pkg/cluster/operator.go

```diff
@@ -100,3 +100,28 @@ func CustomResourceDefinitionExists(ctx context.Context, cli client.Client, crdG
 
 	return err
 }
+
+// return true if not found, return false if not found.
+// checks on both CRD API version also if it is under deletion.
+func CRDVersioNotExists(ctx context.Context, cli client.Client, crdGK schema.GroupKind, version string) (bool, error) {
```

Comment:
I usually then to use the positive version, like `HasCRDVersion` as it is usually clear